internal/exchange/websocket/README.md

```diff
@@ -1,10 +1,10 @@
-# GoCryptoTrader Exchange Stream Package
+# GoCryptoTrader Exchange Websocket Package
```

Comment:
My last request is to graduate this readme into its own true template. Please see my diff.
https://gist.github.com/gloriousCode/3e0a2444b480c1c69b26c2b9a510c635

Reply:
Getting a corrupt patch message from that:
```
❯ curl https://gist.githubusercontent.com/gloriousCode/3e0a2444b480c1c69b26c2b9a510c635/raw/b21629ccd61c036480d0c8a6340868b1f27d206b/diff.diff | git apply
error: corrupt patch at line 217
```
Just looking into why.

Reply:
I think maybe it's the way I make them, shazbert has also said my diffs suck. I just call `git diff > diff.diff` and copy its contents

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/48594292eb

Reply:
> I think maybe it's the way I make them, shazbert has also said my diffs suck. I just call `git diff > diff.diff` and copy its contents
I use: `git diff --patch | pbcopy` to put it in clipboard.
But that should be the same as what you've done. Maybe trailing spaces. I haven't looked too much into it.

Reply:
❤️ It's not how you diff, but what's inside your diff that matters ❤️

Reply:
`git apply --reject --whitespace=fix diff.diff` can be used if the line endings have issues, since copying the contents will lose the last line ending